src/sentry/snuba/discover.py

```diff
@@ -67,15 +67,19 @@ def is_real_column(col):
     return True
 
 
-def find_reference_event(reference_event):
+def find_reference_event(reference_event, real=True):
```

Comment:
This is a nit but `real` is not a great argument name. Is that opposed to a `fake` call?